modern-business/style-editor.scss

```diff
@@ -944,13 +944,24 @@ ul.wp-block-archives,
 
 /* Remove 100% width on figure to center align logo in editor on smaller screens */
 .block-editor-block-list__layout  {
-    .block-editor-block-list__block[data-align="full"] > .block-editor-block-list__block-edit  {
-	    figure.fse-site-logo {
-	        width: inherit;
-        }
+	.block-editor-block-list__block[data-align="full"] > .block-editor-block-list__block-edit  {
+		figure.fse-site-logo {
+			width: inherit;
+		}
 	}
 }
 
+.site-header .wp-block[data-align="full"] {
+	left: calc( -12.5% - 12px );
+	max-width: calc( 125% + 114px );
+	width: calc( 125% + 114px );
```

Comment:
![source](https://user-images.githubusercontent.com/233601/62884012-dc4f8680-bd0b-11e9-90a2-b4fef43f4679.gif)

Reply:
Hah I was thinking of this one:

![calc](https://user-images.githubusercontent.com/1270189/62884118-96ce9180-bcea-11e9-9c93-eabdeb4785c0.gif)

Reply:
What @gwwar said!

Reply:
that gif is exactly how I felt trying to wrangle this for the content area 😆